ios/FluentUI/Core/Theme/Tokens/ControlTokens.swift

```diff
@@ -8,7 +8,9 @@ import Foundation
 /// Base class for all Fluent control tokenization.
 open class ControlTokens: NSObject {
     required public override init() {}
+
+    public var globalTokens: GlobalTokens { fluentTheme.globalTokens }
```

Comment:
size still reasonable?

Reply:
Good question, I'll check. But hopefully two vars isn't tooooo bad.

Reply:
Just checked: making these public adds 176 bytes to our binary, which I think I can live with!